### ISSUE TITLE
Changing number of epochs 

### ISSUE THREAD

**Issue by ananv21** [RUCAIBox/UniSRec#13]
Hello,
I'm trying to run the UniSRec model from scratch (using 'python finetune.py -d Scientific') and I want to change the number of epochs. How can I do this?

Thanks!

**Comment by hyp1231** [comment]
Hi, thanks for your interest!

You can use `python finetune.py -d Scientific --epochs=50` to change the maximum number of training epochs. In default, `epochs` is 300.

Or you can use `python finetune.py -d Scientific --stopping_step=30` to change the early stopping. If `stopping_step` is 30, it means that the training may stop earlier if the validation metric does not update (to a better result) for 30 epochs. In default, `stopping_step` is 10.

More details and parameters can be found in the [API doc](https://recbole.io/docs/user_guide/config/training_settings.html) of RecBole
